app/controllers/shopify_app/app_proxy_controller.rb

```diff
@@ -0,0 +1,9 @@
+module ShopifyApp
+  class AppProxyController < ApplicationController
+    include ShopifyApp::AppProxyVerification
+
+    skip_before_action :verify_authenticity_token
+
+    before_action :verify_proxy_request
+  end
+end
```

Comment:
including the concern matches how we've done the other controller types. For webhooks I believe we offer the concern separately and then recommend overwriting the controller if required.